src/form/input/Input.tsx

```diff
@@ -56,6 +56,7 @@ export type InputProps = Omit<
     locale?: string;
     maximumFractionDigits?: number;
   };
+  inputRef?: React.RefObject<HTMLInputElement>;
```

Comment:
Let's use `forwardRef` instead. This will allow this component to take the ref that it receives as a prop and then we would forward this ref to the `input` element within this component.